src/uct/ib/ud/base/ud_ep.c

```diff
@@ -402,6 +406,7 @@ uct_ud_ep_process_ack(uct_ud_iface_t *iface, uct_ud_ep_t *ep,
 
     ucs_arbiter_group_schedule(&iface->tx.pending_q, &ep->tx.pending.group);
 
+    ep->tx.slow_tick = uct_ud_slow_tick();
```

Comment:
maybe would be better to remember then "backoff" multiplier instead of "slow_tick", so resetting the backoff would be simply assigning 1.0 instead of floating point multiplication?